frontend/src/components/Profile/Description.component.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Fragment } from 'react';
+
+const description = {
+  background: '#F2F5F7',
+  textAlign: 'left',
+  padding: '1rem 0'
+};
+
+export default function Description({ txt }) {
+  return (
+    <Fragment>
```

Comment:
Unessecary fragment, can just return the p tag

Reply:
Very true!